lib/maze/schemas/trace_validator.rb

```diff
@@ -40,6 +45,61 @@ def validate
         )
       end
 
+      def check(date)
```

Comment:
This doesn't appear to be used, and has an ambiguous name, could it be removed or renamed?